app/backend/templates/host_reference.md

```diff
@@ -1,5 +1,5 @@
 ---
-subject: "You've received a reference!"
+subject: "You've received a reference from {{ escape(reference.from_user.name) }}!"
```

Comment:
Do we want to specify kind of reference? We specify in friend reference.

Reply:
I think it's fine, let's wait for proper designs and a proper copy.